Tests/run_tests.py

```diff
@@ -148,7 +148,7 @@ def is_numpy():
                             "Bio.PDB.Polypeptide",
                             "Bio.PDB.Selection",
                             "Bio.SeqIO.PdbIO",
-                            "Bio/SVDSuperimposer/__init__.py"
+                            "Bio/SVDSuperimposer/__init__"
```

Comment:
This should be `` "Bio.SVDSuperimposer",`` (import namespace with dots, not filename with slashes).

Also include the trailing comma even though this is currently the last entry in the list (so that as here, next time we add something to the list it is a one line insertion).